policies/conflict-of-interest.md

```diff
@@ -57,33 +58,21 @@ It is important to note that the existence of a conflict does not mean the trans
-
-The CFC shall ascertain that all material facts regarding the transaction and the covered individual’s conflict of interest have been disclosed to the CFC and shall compile appropriate data, such as comparability studies, to determine fair market value for the transaction.
-
-After exercising due diligence, which may include investigating alternatives that present no conflict, the CFC shall determine whether the transaction is in the Foundation’s best interest, for its own benefit, and whether it is fair and reasonable to the Foundation; the majority of disinterested members of the CFC then in office may approve the transaction.
+Upon the disclosure of a potential conflict, the CFC shall determine if a conflict of interest exists. The CFC may ask questions of and receive presentation(s) from the covered individual(s) and any other interested person(s), but shall deliberate and vote on the transaction in their absence.
 
 ### Records of Review
 
-The minutes of any meeting of the CFC pursuant to this policy shall contain the name of each covered individual who disclosed or was otherwise determined to have an interest in a transaction; the nature of the interest and whether it was determined to constitute a conflict of interest; any alternative transactions considered; the members of the CFC who were present during the deliberations on the transaction, those who voted on it, and to what extent interested persons were excluded from the deliberations; any comparability data or other information obtained and relied upon by the CFC and how the information was obtained; and the result of the vote, including, if applicable, the terms of the transaction that was approved and the date it was approved.
+The minutes of any CFC meeting that addresses conflict of interest disclosure or resolution will include:
+
+- The names of individuals with a potential conflict, the nature of their interest, and whether it was considered a conflict.
+- Details on alternative options reviewed, the CFC members who deliberated and voted, including how conflicts of interest were managed during these discussions.
+- The outcome of the vote, specifying the transaction's terms and approval date if relevant.
 
 #### Annual Disclosure and Compliance
 
-- CFC members (Councilors), EGC Members, and key employees (if any) must annually affirm that they have received, read, understood this policy, and that they agree to comply with it. They must disclose any financial interests and family relationships that could give rise to conflicts of interest.
+- CFC members (Councilors), EGC Members, and key employees (if any) must annually affirm that they have received, read, understood this policy, and that they agree to comply with it.
 - General CF members must affirm that they have received, read, understood this policy, and that they agree to comply with it, when applicable.
 
-## Violations
```

Comment:
What is the process if a violation is found?

Reply:
It should be handled as we would any other violation (which is in the bylaws). We either keep that information in the bylaws (suspension/removal/disciplinary action), or we repeat it in each policy.